### DISCUSSION TITLE
Welcome to fuzzylogic Discussions!

### DISCUSSION THREAD

**Discussion by amogorkon** [Python-Fuzzylogic/fuzzylogic#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PeterVermaas** [comment]
Hi, Currently new and appreciate your work done to make it work! I couldn't find an overview of functions available (lets say available args). Is there a kind of document available with the required args?

**Reply by amogorkon** [reply]
Hi, sorry for the late response.
Reminds me, I did set up a Readthedocs job, but haven't checked it in a long while, I really should get back to that, thanks!
In the meantime there's the [Showcase](https://github.com/amogorkon/fuzzylogic/blob/master/docs/Showcase.ipynb) as an overview.
